src/main/java/com/revature/services/QuestionLogicImpl.java

```diff
@@ -18,8 +20,25 @@
 	
 	@Autowired
 	private QuestionRepository dao;
+	
+	@Autowired
+	private QuestionCommentRepository commentDao;
+	
+//CREATE-----------------------------------
+	@Override
+	@Transactional
+	public QuestionPool createQuestion(QuestionPool question) {
+		return dao.save(question);
+	}
 
 	@Override
+	@Transactional
+	public QuestionComment createComment(QuestionComment comment) {
+		return commentDao.save(comment);
+	}
+
+//RETRIEVE---------------------------------
+	@Override
 	public Page<QuestionPool> getAllQuestions(Pageable pageable, String subject) {
 		
 		if(subject.equalsIgnoreCase("all")){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the "all" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1132)